sos/report/plugins/openstack_aodh.py

```diff
@@ -33,13 +33,13 @@ def setup(self):
 
         if self.get_option("all_logs"):
             self.add_copy_spec([
-                "/var/log/aodh/*",
-                "/var/log/httpd/aodh*",
```

Comment:
Same comment like to `collectd` plugin.

Reply:
same here: the only version using /var/log/... was OSP 10, which is out of support since Dec 2021, see https://access.redhat.com/support/policy/updates/openstack/platform/non-current

Reply:
Since we're doing the override to `/var/log/containers` in the `RedHatOpenStackAodh` class, these should be left as their originals, yes?

Reply:
Sorry for the noise, reading through this again.

We're changing it to the `/var/log/containers` path here for the base plugin, which means for *all* distributions. We're then _again_ specifying it in the RH-specific plugin. So we have the same collection defined in two spots now.

We either would have the `/var/log/containers` collection in the base plugin and *not* have the override in `RedHatOpenStackAodh.setup()`, _or_ we would have the base plugin stay on the "old" `/var/log` paths and keep the override in `setup()`.

Without acks from other distros, I'd say we should do the latter. Have the `setup()` override like you have now, but keep the old paths in the base plugin so we don't inadvertently break collections on non-RH environments that aren't using a containerized setup.